frontend/app/forms/FeedbackForm.scala

```diff
@@ -0,0 +1,36 @@
+package forms
+
+import play.api.data.Forms._
+import play.api.data.format.Formatter
+import play.api.data.{Form, FormError, Mapping}
+import model.FeedbackType
+
+/**
+  * Created by alex_ware on 20/01/2017.
+  */
+object FeedbackForm {
+  case class FeedbackForm(category: FeedbackType, page: String, feedback: String, name: String, email: String)
```

Comment:
This should be a companion case class to  the `FeedbackForm` object! ie bring this case class up one level